packages/kbn-ftr-common-functional-ui-services/services/all.ts

```diff
@@ -6,8 +6,14 @@
  * Side Public License, v 1.
  */
 
+import { RetryService } from '@kbn/ftr-common-functional-services';
 import { RetryOnStaleProvider } from './retry_on_stale';
+import { RemoteProvider } from './remote';
+import { FindProvider } from './find';
 
 export const services = {
   retryOnStale: RetryOnStaleProvider,
+  __webdriver__: RemoteProvider,
+  find: FindProvider,
+  retry: RetryService,
```

Comment:
With the `retry` service added here, we're effectively exporting it from `@kbn/ftr-common-functional-services` and `@kbn/ftr-common-functional-ui-services`, which is not ideal. I think we should separate the services that we need for the package internal FTR provider context (where the retry service is needed) and the services that we actually export from this UI sepcific package. Right now, they are the same (everything in `all.ts`).